app/code/Aspire/Module/etc/adminhtml/system.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0"?>
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:module:Magento_Config:etc/system_file.xsd">
+    <system>
+        <tab id="aspire" translate="label" sortOrder="10">
+            <label>Aspire Module</label>
+        </tab>
+        <section id="module" translate="label" sortOrder="130" showInDefault="1" showInWebsite="1" showInStore="1">
```

Comment:
Why you starting the order from "130". Any reason behind this.

Reply:
I have updated to one. Only for sort order I have added random number